### PR TITLE
implement module geolocation in project.

### PR DESCRIPTION
**Pull request by LeonardoPC23**
Implement permiss in manifest
implement module geolocation in project.

### CODE REVIEW

File: geolocation/build.gradle [hunk 1]
```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion 29
```
Comment (antoniunix) [inline geolocation/build.gradle:6]: you must use the dependencies.gradle

File: geolocation/build.gradle [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    buildToolsVersion "29.0.2"
+
+
+    defaultConfig {
+        applicationId "com.chuys.gshp.geolocation"
+        minSdkVersion 15
```
Comment (antoniunix) [inline geolocation/build.gradle]: same case, use de dependencies.gradle file please

File: geolocation/src/main/java/com/chuys/gshp/geolocation/MainActivity.kt [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+        setSupportActionBar(toolbar)
+
+        fab.setOnClickListener { view ->
```
Comment (antoniunix) [inline geolocation/src/main/java/com/chuys/gshp/geolocation/MainActivity.kt]: please use a naming convention

File: geolocation/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+                android:layout_height="?attr/actionBarSize"
+                android:background="?attr/colorPrimary"
+                app:popupTheme="@style/AppTheme.PopupOverlay" />
+
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <include layout="@layout/content_main" />
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/fab"
```
Comment (antoniunix) [inline geolocation/src/main/res/layout/activity_main.xml]: use naming convention please

**Review by antoniunix — CHANGES_REQUESTED** [review]

File: geolocation/build.gradle (hunk 1 above)
Comment (AGET) [reply geolocation/build.gradle:6]: Podemos meterlo con variable lo de la version 29

File: geolocation/build.gradle [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        targetSdkVersion 29
+        versionCode 1
+        versionName "1.0"
```
Comment (AGET) [inline geolocation/build.gradle]: We should put all values ​​through variables

File: geolocation/build.gradle [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'androidx.appcompat:appcompat:1.0.2'
+    implementation 'androidx.core:core-ktx:1.0.2'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
+    implementation 'com.google.android.material:material:1.0.0'
+    implementation 'com.google.android.gms:play-services-location:17.0.0'
+    testImplementation 'junit:junit:4.12'
+    androidTestImplementation 'androidx.test:runner:1.1.1'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
```
Comment (AGET) [inline geolocation/build.gradle]: same comment, this to take a better control

**Review by AGET — CHANGES_REQUESTED** [review]
If the changes can be made it would be great

**Pull request closed without merging**